recipe/meta.yaml

```diff
@@ -23,12 +24,11 @@ requirements:
   run:
     - python
     - google-auth >=1.25.0,<3.0dev
-    - google-api-core >=1.29.0,<3.0dev
-    - google-cloud-core >=1.6.0,<3.0dev
-    - google-resumable-media >=1.3.0,<3.0dev
+    - google-api-core >=1.31.5,<3.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
+    - google-cloud-core >=2.3.0,<3.0dev
+    - google-resumable-media >=2.3.2
     - requests >=2.18.0,<3.0.0dev
-    - protobuf
-    - six
+    - protobuf <5.0.0dev
```

Comment:
Protobuf is an extras dependency, https://github.com/googleapis/python-storage/blob/aee58b2b0eeafe3e26090df70951a5688148befb/setup.py#L37. Could you add a comment like conda-forge did? See https://github.com/conda-forge/google-cloud-storage-feedstock/blob/0b29f40b1aff6e92d6e130e0f503c91e7731117e/recipe/meta.yaml#L28